src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final int DIVIDER_FOR_AREA = 2;
+    private int bothside;
+    private int height;
+
+    public Rectangle(String color, int bothside, int height) {
```

Comment:
The constructor should not begin with an empty line. Please remove the empty line above the constructor to follow the best practices mentioned in the checklist.